src/components/picker/picker.tsx

```diff
@@ -436,7 +435,7 @@ export class Picker {
         this.handleSearchResult(query, result);
     }
 
-    private handleChange(event: CustomEvent<Chip | Chip[]>) {
+    private handleRemoveChip(event: CustomEvent<Chip | Chip[]>) {
```

Comment:
This does not make sense to rename :smile: 
This will be called when a chip is added as well, `handleChangeChip` would be better name if you want to rename it :slightly_smiling_face: